benchmarks/src/main.rs

```diff
@@ -86,6 +92,52 @@ const GRIDFS_UPLOAD_BENCH: &'static str = "GridFS upload";
 const LDJSON_MULTI_IMPORT_BENCH: &'static str = "LDJSON multi-file import";
 const GRIDFS_MULTI_UPLOAD_BENCH: &'static str = "GridFS multi-file upload";
 
+#[derive(Copy, Clone, num_enum::TryFromPrimitive, PartialEq, Eq, Hash, PartialOrd, Ord)]
+#[repr(u8)]
+enum BenchmarkId {
```

Comment:
the previous approach of using a vec of bool values was very error prone and hard to extend, so I updated it to use an enum instead.